src/jdk.jlink/linux/classes/jdk/tools/jlink/internal/plugins/StripNativeDebugSymbolsPlugin.java

```diff
@@ -131,6 +136,33 @@ public ResourcePool transform(ResourcePool in, ResourcePoolBuilder out) {
         return out.build();
     }
 
+    /**
+     * Method to determine if a particular resource should be stripped.
+     *
+     * Particular paths are added here to handle libraries within the openjceplus module.
+     * The FIPS certified library located in the C/icc directory is sensitive to
+     * any modifications to the native library. Performing any modifications to the library
+     * in any way, causes the FIPS library to fail to load due to a self verification check made.
+     *
+     * @param resource the resource to examine for stripping eligibility.
+     * @return return true if stripping should be done on a particular resource, false otherwise.
```

Comment:
Please omit the trailing periods from these tags.

Reply:
Trailing periods removed from @param and @return phrases.